webapp/tests/karma/ts/modules/contacts/contacts-content.component.spec.ts

```diff
@@ -138,8 +144,10 @@ describe('Contacts content component', () => {
           { provide: ModalService, useValue: modalService },
           { provide: ResponsiveService, useValue: responsiveService },
           { provide: ContactMutedService, useValue: contactMutedService },
+          { provide: TelemetryService, useValue: telemetryService },
```

Comment:
Since we are stubbing out the `SearchTelemetryService`, I would not expect anything to be hitting the `TelemetryService`?